bot/utilities/utils/config.py

```diff
@@ -0,0 +1,97 @@
+# Standard Library
```

Comment:
Does this file need a license header from r.danny? not sure so lmk

Reply:
It will, yes.